packages/replay-next/components/sources/SourceListRow.tsx

```diff
@@ -168,7 +168,7 @@ const SourceListRow = memo(
       if (tokens) {
         for (let index = 0; index < tokens.length; index++) {
           const token = tokens[index];
-          if (token.columnIndex + token.value.length > columnIndexEnd) {
+          if (token.columnIndex >= columnIndexEnd) {
```

Comment:
I'm not sure what I was thinking originally, but this _seems_ like a more appropriate termination check. Note that we'd never expect to break _within_ a token because of how column breakpoints work.